src/main/java/org/commcare/formplayer/repo/impl/PostgresFormSessionRepo.java

```diff
@@ -42,36 +42,13 @@
     private String purgeConfig;
 
     @PostConstruct
-    public void purgeFormSessions() {
-        // Modeled on https://stackoverflow.com/a/6730401/2820312
-        try {
-            String createFuncQuery = "create or replace function custom_safe_cast(text,anyelement) \n" +
-                    "returns anyelement \n" +
-                    "language plpgsql as $$ \n" +
-                    "begin \n" +
-                    "    $0 := $1; \n" +
-                    "    return $0; \n" +
-                    "    exception when others then \n" +
-                    "        return $2; \n" +
-                    "end; $$;";
-            this.jdbcTemplate.execute(createFuncQuery);
-            String deleteQuery = replaceTableName(
-                    "delete from %s where custom_safe_cast(dateopened, '2011-01-01'::timestamp) < NOW() - INTERVAL '7 days';"
-            );
-            if ("startup".equals(purgeConfig)) {
-                log.info("Beginning state form session purge");
-                long start = System.currentTimeMillis();
-                int deletedRows = this.jdbcTemplate.update(deleteQuery);
-                long elapsed = System.currentTimeMillis() - start;
-                log.info(String.format("Purged %d stale form sessions in %d ms", deletedRows, elapsed));
-            } else {
-                log.debug(String.format("Skipping stale session purge due to current trigger " +
-                        "config '%s'. To enable, set " +
-                        "commcare.formplayer.purge.trigger=startup", purgeConfig));
-            }
-        } catch (Exception e) {
-            // Don't crash for this. Not fatal and prevents start-up
-            log.error("Exception purge form sessions", e);
+    public void purge() {
+        if ("startup".equals(purgeConfig)) {
```

Comment:
I don't think we need to retain this once we replace it with the scheduled runs. This was just a stopgap to leave us an option to trigger the process if needed, so we can remove it with this PR.